external-crates/move/crates/move-borrow-graph/src/references.rs

```diff
@@ -209,7 +209,7 @@ impl<Loc: Copy, Lbl: Clone + Ord> PartialEq for BorrowEdge<Loc, Lbl> {
 
 impl<Loc: Copy, Lbl: Clone + Ord> Eq for BorrowEdge<Loc, Lbl> {}
 
-#[allow(clippy::incorrect_partial_ord_impl_on_ord_type)]
+#[allow(clippy::non_canonical_partial_ord_impl)]
```

Comment:
What's it complaining about here?

Reply:
Or is this new?

Reply:
I ran `move-clippy` and this gets complained on as well, so it is new.